LifeHelper.Services/Areas/User/Validators/UserInputValidator.cs

```diff
@@ -0,0 +1,29 @@
+﻿using FluentValidation;
+using LifeHelper.Services.Areas.User.DTOs;
+
+namespace LifeHelper.Services.Areas.User.Validators;
+
+public class UserInputValidator : AbstractValidator<UserInputDto>
+{
+    public UserInputValidator()
+    {
+        RuleFor(x => x.Nickname)
+            .NotEmpty()
+                .WithMessage("{PropertyName} is required!")
+            .Length(3, 21)
+                .WithMessage("{PropertyName} must contain from {MinLength} to {MaxLength} characters!");
+        
+        RuleFor(x => x.Email)
+            .NotEmpty()
+                .WithMessage("{PropertyName} is required!")
+            .EmailAddress()
+                .WithMessage("Incorrect Email address!");
+
+        RuleFor(x => x.Password)
+            .NotEmpty()
+                .WithMessage("{PropertyName} is required!")
+            .CheckPassword();
+        
```

Comment:
upper, you use .EmailAddress for checking email
so, use .Password to chek password